package.json

```diff
@@ -39,7 +39,7 @@
     "react-native-map-link": "^1.0.6",
     "react-native-maps": "^0.21.0",
     "react-native-permissions": "^1.1.1",
-    "react-native-safe-area-view": "robbiemccorkell/react-native-safe-area-view",
+    "react-native-safe-area-view": "^0.9.0",
```

Comment:
There is one more thing that needs removing later in this file on the `resolutions` property.

Reply:
resolutions removed, pushed up 👍